tests/e2e/.eslintrc.js

```diff
@@ -1,5 +1,9 @@
+const rulesDirPlugin = require( 'eslint-plugin-rulesdir' );
+rulesDirPlugin.RULES_DIR = 'tests/e2e/wc-blocks-eslint-rules';
```

Comment:
My IDE wouldn't work unless I changed this to
```suggestion
rulesDirPlugin.RULES_DIR = 'wc-blocks-eslint-rules';
```

Reply:
Thank you for review this, @opr.

> My IDE wouldn't work unless I changed this to

Hmm, this is interesting, according to the [plugin's documentation](https://github.com/not-an-aardvark/eslint-plugin-rulesdir#usage):

> The paths are resolved from the current working directory, and indicates where you would like the plugin to load your rules from.

I think WebStorm must be loading the eslint config from a different path than VS Code does. I just changed the dir to: 

```
rulesDirPlugin.RULES_DIR = `${ __dirname }/wc-blocks-eslint-rules`;
```

This way we get the absolute path for the dir where the eslint file is executing and hopefully this will work for multiple editors. Let me know if this fixes the issue for you as well.